x11-libs/gtksourceview/gtksourceview3-3.24.0.recipe

```diff
@@ -0,0 +1,101 @@
+SUMMARY="A text widget implementing syntax highlighting and other features"
+DESCRIPTION="GtkSourceView is a GNOME library that extends GtkTextView, the standard GTK \
+widget for multiline text editing. GtkSourceView adds support for syntax \
+highlighting, undo/redo, file loading and saving, search and replace, a \
+completion system, printing, displaying line numbers, and other features \
+typical of a source code editor."
+HOMEPAGE="https://wiki.gnome.org/Projects/GtkSourceView/"
+COPYRIGHT="Christian Hergert
+Sebastien Wilmet"
+LICENSE="GNU LGPL v2.1"
+REVISION="2"
+SOURCE_URI="https://gitlab.gnome.org/GNOME/gtksourceview/-/archive/$portVersion/gtksourceview-$portVersion.tar.gz"
+SOURCE_DIR="gtksourceview-$portVersion"
+CHECKSUM_SHA256="46cd7be87df194473a7ed26af486478f9b3521fda7fb7537f6ad2951ac5eb86d"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gtksourceview3$secondaryArchSuffix = $portVersion compat >= 3
+	lib:libgtksourceview_3.0$secondaryArchSuffix = $portVersion compat = 3
```

Comment:
LibVersion is 1.8.0, please adjust/use libVersion/libVersionCompat
EDIT: this version was for 3.24.11, maybe it's different on 3.24.0, please check.